src/shared/lib/markDownRules.tsx

```diff
@@ -546,8 +560,44 @@ const getContainerAlignTag = (parents: ASTNode[]): AlignmentTag | undefined => {
   return tag;
 };
 
-export const preprocessImageLinks = (content: string) => {
-  return content?.replace(/(!\[.*\]\s*\(.*?) =\d*x\d*(\))/g, '$1$2');
-};
+export const preprocessImageLinks = (content: string): string => {
```

Comment:
I'm still generally against the notion of modifying the url, and would instead prefer to use markdown-it-imsize to help determine dimensions without modifying content. Since images are cached I believe this can be handled in the image node instead without major changes in your code.